contracts/governor/src/storage.rs

```diff
@@ -7,8 +7,8 @@ const VOTER_TOKEN_ADDRESS_KEY: &str = "Votes";
 const SETTINGS_KEY: &str = "Settings";
 const IS_INIT_KEY: &str = "IsInit";
 const PROPOSAL_ID_KEY: &str = "ProposalId";
-pub(crate) const LEDGER_THRESHOLD_SHARED: u32 = 172800; // ~ 10 days
-pub(crate) const LEDGER_BUMP_SHARED: u32 = 241920; // ~ 14 days
```

Comment:
We should keep the shared data at 10/14 days respectively. This includes things like the contract code, etc.

We should create another set of constants that represents 30/31 days for the data for a proposal (or other things that might need longer to live)